Login_Form/serializers.py

```diff
@@ -36,3 +36,12 @@ class TaskUpdateSerializer(serializers.Serializer):
     username = serializers.CharField(required=True)
     password = serializers.CharField(required=True)
     task_status=serializers.CharField(required=True)
+class CommentsSerializer(serializers.Serializer):
```

Comment:
why are you using M, E username?

Reply:
to differntiate while assigning task by manager to employee in postman that which username is of which role